pkg/thing/mocks/publisher.go

```diff
@@ -0,0 +1,29 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakePublisher struct {
+	mock.Mock
+	ReturnErr error
+	SendError error
+	Token     string
+}
+
+func (fp *FakePublisher) SendRegisterDevice(msg network.RegisterResponseMsg) error {
+	ret := fp.Called(msg)
+	return ret.Error(0)
+}
+
+func (fp *FakePublisher) SendUpdatedSchema(thingID string) error {
+	ret := fp.Called(thingID)
+	return ret.Error(0)
+}
+
+func (fp *FakePublisher) SendThings(things []*entities.Thing) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method FakePublisher.SendThings should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)